debian/curtin_userdata_custom_amd64

```diff
@@ -16,8 +16,7 @@ late_commands:
   maas: [wget, '--no-proxy', '{{node_disable_pxe_url}}', '--post-data', '{{node_disable_pxe_data}}', '-O', '/dev/null']
   late_1: mount --bind $TARGET_MOUNT_POINT /mnt
   late_2: grep -A2 datasource /etc/cloud/cloud.cfg.d/91_kernel_cmdline_url.cfg | sed 's/curtin//' | tee /mnt/etc/cloud/cloud.cfg.d/debian.cfg
-  late_3: sed -i 's@ubuntu.com/ubuntu@debian.org/debian@g;s@archive@deb@g;s@ubuntu@debian@g;s@Ubuntu@Debian@g;' /mnt/etc/cloud/cloud.cfg
-  late 4: sed -i 's/security.debian.org\/debian/security.debian.org/g' /mnt/etc/cloud/cloud.cfg
+  late_3: sed -i 's@ubuntu.com/ubuntu@debian.org/debian@g;s@archive@deb@g;s@ubuntu@debian@g;s@Ubuntu@Debian@g;' /mnt/etc/cloud/cloud.cfg && sed -i 's/security.debian.org\/debian/security.debian.org/g' /mnt/etc/cloud/cloud.cfg
```

Comment:
```suggestion
  late_3: sed -i 's@ubuntu.com/ubuntu@debian.org/debian@g;s@archive@deb@g;s@ubuntu@debian@g;s@Ubuntu@Debian@g;s@security.debian.org/debian@security.debian.org@g' /mnt/etc/cloud/cloud.cfg
```

Let's do it all in one invocation of sed

Reply:
Tested and confirmed this works as expected